src/packages/name-service/blockstack-service.ts

```diff
@@ -186,7 +186,7 @@ export class BlockstackService extends nameService.NameService {
             throw ErrorHelper.getPackageError(PackageErrorCode.CouldNotFindKeyPairToRegisterName);
         }
 
-        await this._gaiaService.uploadProfileInfo(identityKeyPair.privKey);
+        await this._gaiaService.uploadProfileInfo(identityKeyPair.privKey, IdTranslator.blockstackDomainToCruxDomain(this._domain));
```

Comment:
this._domain should be renamed to this._blockstackdomain

Reply:
I will be addressing this change in the upcoming PR.